scripts/charts.sh

```diff
@@ -134,8 +134,8 @@ run_pull() {
 
   mkdir -p "${CHARTS}/tmp"
 
-  error="$(helm pull "${chart}" --version "${requested_version}" --untar --untardir "${CHARTS}/tmp/${chart%%/*}" 2>&1 || true)"
-
+  error="$(helm pull "${chart}" --version "${requested_version}" --untar --untardir "${CHARTS}/tmp/${chart%%/*}" 2>&1 > /dev/null || true)"
```

Comment:
Does this mean that it redirects all to `/dev/null` or do `stderr` still point to whatever `stdout` pointed to before sending it to `/dev/null`?

Reply:
`stderr` will be redirected to where `stdout` was redirected before

Reply:
This is kind of obvious when you think about it but I had never thought about it so I just wanted to prove it to myself:
```bash
$ { echo stdout; echo stderr 1>&2; }
stdout
stderr
$ { echo stdout; echo stderr 1>&2; } 2>&1 >/dev/null
stderr
$ { echo stdout; echo stderr 1>&2; } >/dev/null 2>&1
$
```
Ordering matters. :)